networkit/cpp/auxiliary/BucketPQ.cpp

```diff
@@ -92,8 +97,9 @@ void BucketPQ::remove(const index& value) {
     }
 }
 
-std::pair<int64_t, index> BucketPQ::extractMin() {
+std::pair<int64_t, index> BucketPQ::extractMin(bool removeElement) {
     if (size() == 0) {
+        WARN("The priority queue is empty!");
```

Comment:
Why does this warn?

It should either be allowed to call this on an empty queue (that's the current state), or it should be a precondition that the queue is non-empty (in which case this should be a hard error).